client/src/app/site/base/base-view.ts

```diff
@@ -27,24 +27,42 @@ export abstract class BaseViewComponent extends BaseComponent implements OnDestr
         super(titleService, translate);
     }
 
+    /**
+     * Opens the snack bar with the given message.
+     * This snack bar will only dismiss if the user clicks the 'OK'-button.
+     */
+    protected raiseWarning = (message: string): void => {
```

Comment:
Where is the difference to `raiseError`?

Reply:
This function calls a snackbar, which does not appear until the user clicks the 'okay'-button.
So, the user notices this.